operations/client.go

```diff
@@ -20,6 +25,52 @@ func Client() cli.Command {
 	}
 }
 
+// RunKanopyOIDCLogin executes the kanopy-oidc login command and captures the JWT token from the output.
+// It also displays output to the user in real-time while ensuring that the JWT token is not printed.
+func RunKanopyOIDCLogin() (string, error) {
+	cmd := exec.Command("kanopy-oidc", "login", "-n", "-f", "device")
+
+	stdout, err := cmd.StdoutPipe()
+	if err != nil {
+		return "", errors.Wrap(err, "creating stdout pipe")
+	}
+
+	cmd.Stderr = os.Stderr // Show stderr output directly to the user
+
+	// Start the command
+	if err := cmd.Start(); err != nil {
+		return "", errors.Wrap(err, "starting kanopy-oidc login")
+	}
+
+	// Process the output - display non-token outputs and capture the token
+	var tokenBuf bytes.Buffer
+	scanner := bufio.NewScanner(stdout)
+
+	for scanner.Scan() {
+		line := scanner.Text()
+
+		// Check if the line contains a JWT (usually starts with "ey")
```

Comment:
im concerened about the "usually" 
maybe we shouldn't check this because maybe jwt tokens can change to not start like this 

(unless there's documentation saying it will stay like this?)

Reply:
Based on the accepted answer [here](https://stackoverflow.com/questions/49517324/why-header-and-payload-in-the-jwt-token-always-starts-with-eyj) it seems to be a pretty safe bet.